libpod/events/filters.go

```diff
@@ -52,7 +52,7 @@ func generateEventFilter(filter, filterValue string) (func(e *Event) bool, error
 			if e.Type != Volume {
 				return false
 			}
-			return strings.HasPrefix(e.ID, filterValue)
```

Comment:
I think we need to keep this check but also compare with the name as done in other filters above.  Otherwise, we are breaking existing filters using short IDs and IDs in general.